src/Lime.Protocol.UnitTests/Serialization/Newtonsoft/EnvelopeSerializerTests.cs

```diff
@@ -13,10 +9,32 @@
 using Newtonsoft.Json.Linq;
 using NUnit.Framework;
 using Shouldly;
-using Lime.Messaging;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
 
 namespace Lime.Protocol.UnitTests.Serialization.Newtonsoft
 {
+    sealed class CustomJsonConverterTest : JsonConverter
```

Comment:
remove test code

Reply:
LOL sorry I hadn't noticed this is in a test project.

A better name might be in order though. Perhaps `DummyJsonConverter`.

Reply:
Or perhaps change the implementation to not throw and do nothing instead and then name it `StubJsonConverter`

Reply:
besides this, LGTM 👍🏻 (though I'm biased to say so 😅)

Reply:
I consider valid and accept the suggestion of the name `DummyJsonConverter` because it makes clear the intent of the class.